packages/test/test-end-to-end-tests/src/test/handleValidation.spec.ts

```diff
@@ -345,4 +429,103 @@ describeCompat("handle validation", "NoCompat", (getTestObjectProvider, apis) =>
 			assert(actualId.ITestFluidObject?.context.id, idA);
 		});
 	}
+
+	for (const detachedDds1Utils of ddsTypes) {
+		for (const detachedDds2Utils of ddsTypes) {
+			for (const attachedDdsUtils of ddsTypes) {
+				it(`stores ${detachedDds1Utils.id} handle in ${detachedDds2Utils.id} and attaches by storing in ${attachedDdsUtils.id}`, async () => {
+					/**
+					 * setup required for all portions of the test
+					 */
+					const { container1, provider, testContainerConfig } = await setup(
+						getTestObjectProvider,
+						apis,
+					);
+
+					const attachedDataStore =
+						(await container1.getEntryPoint()) as ITestFluidObject;
+
+					const dataStoreB =
```

Comment:
i don't think we want a new data store here and we should just use the attachedDataStore's runtime. The case with a new datastore is interesting, but i think that is covered in the pervious, at least to some degree. The differenence is if the to whole data store is detached, it will get one big summary of the whole datastore and send an attach op for that, but if the datastore is already attached, then each dds will get its own attach op.

Reply:
Something interesting about this - with the new data store, the majority of the test cases fail, except for the ones that use map and directory (the ones that have the bindHandles workaround for AB#7452). Without this data store, they all pass, but I'm not sure if we still need coverage for not binding handles from all of the other DDSs.

Reply:
interesting. it might be useful to have both tests then, those that use a new datastore, and those that do not.